targets/PandABox/scripts/build_top.tcl

```diff
@@ -13,7 +13,7 @@ set_param board.repoPaths $TARGET_DIR/configs
 
 # Create project
 create_project -force -in_memory panda_carrier_top \
-    $BUILD_DIR/panda_carier_top -part xc7z030sbg485-1
+    $BUILD_DIR/panda_carrier_top -part xc7z030sbg485-1
```

Comment:
This typo should be corrected in master before making this merge

Reply:
changes removed and merged from master